content/en/dora_metrics/deployments/_index.md

```diff
@@ -139,6 +139,7 @@ DORA Metrics for the service `shopist` only consider the Git commits that includ
 
 - Change lead time stage breakdown metrics are only available for GitHub.
 - Change lead time is not available for the first deployment of a service that includes Git information.
+- If commits on a feature branch are squashed into a single commit prior to reaching the default branch (meaning that commits are squashed when a pull request is merged into the default branch), the commit history for that pull request is not included in the change lead time calculation. The first commit included in the calculation is the newly created single commit. 
```

Comment:
I would remove the part between parenthesis, as also the Pull Request concept is GitHub-specific whereas squashing and merging is generally available in different git providers (e.g. [GitLab](https://docs.gitlab.com/ee/user/project/merge_requests/squash_and_merge.html))

```suggestion
- If commits on a feature branch are squashed into a single commit prior to being merged into the default branch, the commit history for that pull request is not included in the change lead time calculation. The first commit included in the calculation is the newly created single commit. 
```